.erb/scripts/electron-rebuild.js

```diff
@@ -1,22 +1,21 @@
 import path from 'path';
 import { execSync } from 'child_process';
 import fs from 'fs';
-import { dependencies } from '../../src/package.json';
-
-const nodeModulesPath = path.join(__dirname, '../../src/node_modules');
+import { dependencies } from '../../build/app/package.json';
+import webpackPaths from '../configs/webpack.paths.js';
 
 if (
   Object.keys(dependencies || {}).length > 0 &&
-  fs.existsSync(nodeModulesPath)
+  fs.existsSync(webpackPaths.appNodeModulesPath)
 ) {
   const electronRebuildCmd =
-    '../node_modules/.bin/electron-rebuild --parallel --force --types prod,dev,optional --module-dir .';
+    '../../node_modules/.bin/electron-rebuild --parallel --force --types prod,dev,optional --module-dir .';
   const cmd =
```

Comment:
How about using `$(yarn bin)/electron-rebuild`?

Reply:
Many of our users migrate to npm immediately after setting up the boilerplate. This change will make it slightly more difficult for those users to migrate